### PR TITLE
GLAD Biodiversity Table Update

### PR DESCRIPTION
**Pull request by 01painadam**
## Overview

Updates the GLAD Biodiversity Widget to use the new summary tables (pre-aggregated by week, and pre-grouped by either adm1 or adm2 ). Makes requests _fast_! 🔥 

No Aesthetic changes.

Note that the summary tables are not yet synced with the current GLAD table, so data will not match when testing

### CODE REVIEW

File: app/javascript/services/alerts.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import axios from 'axios';
-import moment from 'moment';
+// import moment from 'moment';
```
Comment (edbrett) [inline app/javascript/services/alerts.js]: dont leave comments please

File: app/javascript/services/alerts.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 
 const REQUEST_URL = `${process.env.GFW_API_HOST_PROD}`;
-const DATASET = process.env.GLAD_PRECALC_DATASET;
+const ISO_DATASET = process.env.ISO_GLAD_SUMMARY;
+const ADM1_DATASET = process.env.ADM1_GLAD_SUMMARY;
```
Comment (edbrett) [inline app/javascript/services/alerts.js]: Can you names these by GLAD_.... instead? So they are grouped together in the alpha in heroku so easier to find.

File: app/javascript/services/alerts.js (hunk 1 above)
Comment (01painadam) [reply app/javascript/services/alerts.js]: My bad!

**Comment by 01painadam** [comment]
Glad tables are now updated with the following id's for the `.env`:

```ISO_GLAD_SUMMARY=391ca96d-303f-4aef-be4b-9cdb4856832c
ADM1_GLAD_SUMMARY=c7a1d922-e320-4e92-8e4c-11ea33dd6e35
ADM2_GLAD_SUMMARY=428db321-5ebb-4e86-a3df-32c63b6d3c83```

**Pull request merged**